src/core/gltf2importer/mikktspace/mikktspace.c

```diff
@@ -0,0 +1,1843 @@
+/** \file mikktspace/mikktspace.c
+ *  \ingroup mikktspace
+ */
+/**
+ *  Copyright (C) 2011 by Morten S. Mikkelsen
+ *
+ *  This software is provided 'as-is', without any express or implied
```

Comment:
I have some concerns about the license here. Since I'm not a lawyer, I've a hard time to figure out if it's really compatible with AGPLv3 in practice. My gut feeling is that it is... but my guts aren't a recognized lawyer.

Reply:
it's the zlib license so I am pretty sure it is fine - it's already used in various Qt 3rdparty dependencies

Reply:
Who should we ask? Looks like is possible, but an opinion from an expert would be nice.

Reply:
Oh right, that's the zlib license. Somehow didn't recognize it. We're fine then.